src/test/java/com/egov/core-services/constants/pgServices.yaml

```diff
@@ -0,0 +1,30 @@
+parameters:
+ invalidTxnAmount: abc
+ txnAmount: 200
+ paidAmount: 500
+ amountGreterThanDue: 340
+ invalidTenantId: ab
+ gateway: AXIS
+ invalidGateway: HDFC
+ invalidCallBackUrl: null
+ consumerCode: PB-PT-2020-12-02-007576
+ invalidConsumerCode: null
+ billId: e30d1915-7408-4624-9e16-ae473202c52b
+ invalidBillId: null
+ nonExistentBillId: abc
+ paidBillId: 953b189a-c61e-44c5-bff2-1b3ea483975a
+ expiredBillId: e4af8ba5-d294-4dd5-9dfb-dd7a4c3eff81
+ productInfo: Common Payment
+ invalidProductInfo: null
+ validTransactionId: PB_PG_2021_01_11_004061_52
+ invalidTransactionId: PG-PG-2020-10-15-002762
+ invalidAuthToken: 123
+ trasanctioIdWhichBillExpired: PB_PG_2020_12_10_001091_67
+ invalidTxnId: abc
+ invalidSearcAuthToken: 123
+ consumerCode: PB-PT-2020-11-04-007372
```

Comment:
please group it as valid and invalid parameters section and remove duplicate values like invalidSearcAuthToken